frontend/dashboard/src/App.js

```diff
@@ -0,0 +1,151 @@
+import React, { useState } from 'react';
+import './styles.css';
+
+function App() {
+  const [selectedTable, setSelectedTable] = useState('Policies');
+
+  return (
+    <div className="dashboard">
+      <Sidebar />
+      <MainContent selectedTable={selectedTable} setSelectedTable={setSelectedTable} />
+    </div>
+  );
+}
+
+function Sidebar() {
+  return (
+    <div className="sidebar">
+      <h1>Life Insurance</h1>
+      <nav>
+        <ul>
+          <li><a href="#">Dashboard</a></li>
+          <li><a href="#">Tax Management</a></li>
+          <li><a href="#">Stock Exchange</a></li>
+        </ul>
+      </nav>
+    </div>
+  );
+}
+
+function MainContent({ selectedTable, setSelectedTable }) {
+  return (
+    <div className="main-content">
+      <Header />
+      <TableButtons selectedTable={selectedTable} setSelectedTable={setSelectedTable} />
+      <TableDisplay selectedTable={selectedTable} />
+    </div>
+  );
+}
+
+function Header() {
+  return (
+    <div className="header">
+      <h2>Hello, user</h2>
+      <button>Sign out</button>
+    </div>
+  );
+}
+
+function TableButtons({ selectedTable, setSelectedTable }) {
+  return (
+    <div className="table-buttons">
+      <TableButtonsBox
+        title="Policies"
+        value="700"
+        color="#4da6ff"
+        active={selectedTable === 'Policies'}
+        onClick={() => setSelectedTable('Policies')}
+      />
+      <TableButtonsBox
+        title="Paid Out"
+        value="230"
+        color="#4da6ff"
+        active={selectedTable === 'Paid Out'}
+        onClick={() => setSelectedTable('Paid Out')}
+      />
+      <TableButtonsBox
+        title="Payment History"
+        value="1000"
+        color="#4da6ff"
+        active={selectedTable === 'Payment History'}
+        onClick={() => setSelectedTable('Payment History')}
+      />
+    </div>
+  );
+}
+
+function TableButtonsBox({ title, value, color, active, onClick }) {
+  return (
+    <div
+      className={`table-buttons-box ${active ? 'active' : ''}`}
+      style={{ borderColor: color }}
+      onClick={onClick}
+    >
+      <h3>{title}</h3>
+      <p>{value}</p>
+    </div>
+  );
+}
+
+function TableDisplay({ selectedTable }) {
+  let data;
+  switch (selectedTable) {
+    case 'Policies':
+      data = mockData.policies;
+      break;
+    case 'Paid Out':
+      data = mockData.paidOut;
+      break;
+    case 'Payment History':
```

Comment:
I'd maybe call it debit order history. It can be confused with the pay out tab

Reply:
unless it's something else? What is payment history? History of pay out and debit orders that were paid, or just like debit orders?

Reply:
I was following the figma, so I'm not too sure but I think it's the debit order history. Will confirm with @Vutlhari-Mashimbyi